src/common/services/mx-communication/mx-communication.module.ts

```diff
@@ -26,7 +27,7 @@ import { MxDataApiService } from './mx-data.service';
     MxIdentityService,
     MxFeedService,
     SlackReportService,
-    MxDataApiService,
+    MxDataApiService, MxToolsService
```

Comment:
keep the same formatter; put to new line

Reply:
done